card/mod/history/lib/card/action/differ.rb

```diff
@@ -45,9 +45,20 @@ def diff_object field, opts
       def content_diff_object opts=nil
         @diff ||= begin
           diff_args = opts || card.include_set_modules.diff_args
-          diff_object :content, diff_args
+          previous = raw_view previous_value(:content), diff_args[:format]
+          current = raw_view value(:content), diff_args[:format]
+
+          Card::Content::Diff.new previous, current, diff_args
         end
       end
+
+      def raw_view content, format=:text
```

Comment:
nice solution. I expected we'd be setting the action_id rather than the content directly, but perhaps this makes more sense here.